src/styles/blocks/calendar.scss

```
@@ -0,0 +1,66 @@
+body {
+  margin: 0;
+}
+
+/* .calendar styles */
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: $days-gap;
+  width: ($day-width * 7) + ($days-gap * 7);
```

```suggestion
  width: ($day-width * 7) + ($days-gap * 6);
```
Между 7 клетками вроде 6 промежутков